### PR TITLE
Fixing some spelling issues in the config.yml

### PR DESCRIPTION
**Pull request by O-Mutt**

### CODE REVIEW

File: config.yml [hunk 1]
```diff
@@ -2,20 +2,20 @@
 script_options:
   # Switch between testnet and mainnet
   # Setting this to False will use REAL funds, use at your own risk
-  TESTNET: True
+  TESTNET: False
```
Comment (O-Mutt) [inline config.yml:5]:
```suggestion
  TESTNET: True
```

File: config.yml [hunk 2]
```diff
@@ -25,17 +25,17 @@ trading_options:
     - DOWN
     - UP
 
-  # Maximum number of coints to hold
-  MAX_COINS: 10
+  # Maximum number of coins to hold
+  MAX_COINS: 20
```
Comment (O-Mutt) [inline config.yml:29]:
```suggestion
  MAX_COINS: 10
```

File: config.yml [hunk 3]
```diff
@@ -25,17 +25,17 @@ trading_options:
...
+  # the amount of time in SECONDS to calculate the difference from the current price
   TIME_DIFFERENCE: 30
 
-  # Numer of times to check for TP/SL during each TIME_DIFFERENCE Minimum 1
+  # Number of times to check for TP/SL during each TIME_DIFFERENCE Minimum 1
   RECHECK_INTERVAL: 6
 
   # the difference in % between the first and second checks for the price.
-  CHANGE_IN_PRICE: 1.25
+  CHANGE_IN_PRICE: 0.8
```
Comment (O-Mutt) [inline config.yml:38]:
```suggestion
  CHANGE_IN_PRICE: 1.25
```

**Comment by ivandardi** [comment]
You could probably also add proper capitalization and the final period for the comments as well.

**Comment by CyberPunkMetalHead** [comment]
@O-Mutt, I'll close this as it's been addressed in @Davidobot 's commit

**Pull request closed without merging**